app/Http/Controllers/IpmaWarningController.php

```diff
@@ -17,7 +19,14 @@ class IpmaWarningController extends Controller
      */
     public function index(): JsonResponse
     {
-        $data = IpmaWarning::query()->where('is_active', 1)->get();
-        return response()->collection($data, new IpmaWarningSerializer());
+        $data = ['data' => []];
+
+        if (Cache::has('ipma_warnings')) {
```

Comment:
Use the `remember()` cache method, instead. See https://laravel.com/docs/5.8/cache

Reply:
humm, the controller will not fetch data bc of the fetcher running queued, so the remember makes no sense here too.